ci/docker/runtime_functions.sh

```diff
@@ -226,15 +226,16 @@ build_arm64() {
     ninja -v
     report_ccache_usage
     export MXNET_LIBRARY_PATH=`pwd`/libmxnet.so
-    cd /work/mxnet/python
-    python setup.py bdist_wheel --universal
-    cp dist/*.whl /work/build
```

Comment:
Did you intend to remove this? We now got the build wheel helper function

Reply:
Thanks, is not needed for android. as there's no python ATM

Reply:
Please correct me if I'm wrong, but I think this is the regular Arm64 script which is used for raspberry pi 3 for example

Reply:
Seems you are right, thanks for noticing.